bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/references/BootstrapBundleJavaScriptReference.java

```diff
@@ -0,0 +1,36 @@
+package de.agilecoders.wicket.core.markup.html.references;
+
+import de.agilecoders.wicket.webjars.request.resource.WebjarsJavaScriptResourceReference;
+
+/**
+ * represents the bootstrap js library
+ *
+ * @author miha
```

Comment:
I am pretty sure `miha` does not care about this anymore :-)

Reply:
Addressed :)